src/app/main/main.component.ts

```diff
@@ -290,6 +287,24 @@ export class MainComponent implements OnInit, OnDestroy {
     return dryRun;
   }
 
+  // 'count' is used to track the number of request. Each request is 10MB, so we are limiting the total to 250 MB.
+  getResults(count: number, projectId: string, inputPageToken: string, location: string, jobId: string)  : Promise<BigQueryResponse> {
+    if (!inputPageToken || count >= 25) {
```

Comment:
Let's put the `25` here into constant in the constants file, maybe `MAX_PAGES = 25`?

Reply:
Done.